scripts/autobotchk

```diff
@@ -276,8 +276,8 @@ if {$systemd} {
   exit
  }
  puts "Creating systemd directory..."
- catch {file mkdir ~/.config/systemd/user } res
- if {[catch {open ~/.config/systemd/user/${botnet-nick}.service w} fd]} {
+ catch {file mkdir -p "$::env(HOME)/.config/systemd/user" } res
```

Comment:
Tcl's file mkdir does not support -p, that's only bash and other shells, Tcl already creates the parent dirs as needed, so we can remove -p

```suggestion
 catch {file mkdir "$::env(HOME)/.config/systemd/user" } res
```